packages/flutter_driver/lib/src/common/fuchsia_compat.dart

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Convenience methods for Flutter application driving on Fuchsia. Can
+/// be run on either a host machine (making a remote connection to a Fuchsia
+/// device), or on the target Fuchsia machine.
+import 'dart:async';
+import 'dart:core';
+import 'dart:io';
+
+import 'package:fuchsia_remote_debug_protocol/fuchsia_remote_debug_protocol.dart';
+
+import 'error.dart';
+
+// TODO(awdavies): Update this to use the hub.
+final Directory _kDartPortDir = Directory('/tmp/dart.services');
```

Comment:
Is this directory something used by the remote_debug_protocol or just a pattern for fuchsia? Ideally we could configure this directory

Reply:
It's a Fuchsia thing.  I'm unsure of a clean way to make this configurable though.

Reply:
Maybe another env variable?  Unsure.

Reply:
How do you invoke flutter driver for fuchsia? We could pass it in as a command line arg through this or the flutter tool.

Reply:
Or I guess this art is running on the device right?

Reply:
Right now we just invoke it as a plain dart application (or a shell script).  Also yeah, this is running on the device.